Projects/Foundations/Schemas/Docs-Topics/D/Divergences/Divergences-Data-Mine/Divergences-data-mine-004-CCI-Divergence.json

```diff
@@ -0,0 +1,130 @@
+{
+    "topic": "Divergences Data Mine",
+    "pageNumber": "4",
+    "type": "CCI Divergence",
+    "definition": {
+        "text": "The CCI Divergence - this indicator analyses highs and lows of a 20 period CCI indicator and compares it to the price action and produces any divergence data found."
+    },
+    "paragraphs": [
+        {
+            "style": "Title",
+            "text": "CCI Divergence on the Charts"
+        },
+        {
+            "style": "Text",
+            "text": "According to investopedia: Momentum indicators are commonly used to smooth out the price action and give a clearer picture. They allow the trader to compare the indicator swings to price swings, rather than having to compare price to price. Momentum is a study of the strength of the price action, it simply checks whether the price and momentum agree or disagree."
+        },        
+		{
+            "style": "Text",
+            "text": "Disagreement between the indicator and price is called divergence, and it can have significant implications for trade management. The amount of agreement/disagreement is relative, so there can be several different patterns that develop in the relationship between price and the indicator."
+        },
+        {
+            "style": "Link",
+            "text": "Understanding momentum - (investopedia)->www.investopedia.com/trading/trading-divergence-and-understanding-momentum/#toc-momentum-divergence"
+        },
+		{
```

Comment:
Did you manually indent the opening brackets to the rigth?
And do you want to delete the last file?

Reply:
For line 14? can't remember but I may have.

Which file are you referring to?

Reply:
Line 17 in the file i highlighted above.

Reply:
Can you regularly app.update?